src/node_api.cc

```diff
@@ -815,24 +782,37 @@ static
 napi_status Unwrap(napi_env env,
                    napi_value js_object,
                    void** result,
-                   v8::Local<v8::Object>* wrapper,
-                   v8::Local<v8::Object>* parent = nullptr) {
+                   bool remove_wrap) {
```

Comment:
Consider using an enum instead of a bool.  More self-descriptive at the call site.